src/modules/shared/components/MetaDataDescription/MetaDataDescription.tsx

```diff
@@ -0,0 +1,51 @@
+import clsx from 'clsx';
+import { FC, useState } from 'react';
+
+import useTranslation from '@shared/hooks/use-translation/use-translation';
+
+import { Blade } from '../Blade';
+import { TextWithNewLines } from '../TextWithNewLines';
+
+import styles from './MetaDataDescription.module.scss';
+
+interface MetaDataDescriptionProps {
+	description: string;
+}
+
+const MetaDataDescription: FC<MetaDataDescriptionProps> = ({ description }) => {
+	const { tText } = useTranslation();
+	const maxLength = 500;
+	const isDescriptionLong = description.length > maxLength;
+	const shortenedDescription = description.substring(0, maxLength);
+	const [isBladeOpen, setIsBladeOpen] = useState(false);
+
+	return (
+		<>
+			<p className="u-pb-24 u-line-height-1-4 u-font-size-14">
+				<TextWithNewLines text={isDescriptionLong ? shortenedDescription : description} />
+				{isDescriptionLong && (
+					<u style={{ cursor: 'pointer' }} onClick={() => setIsBladeOpen(true)}>
```

Comment:
Not a fan of inline styling, especially when there's already a `.scss` file. If it's always done this way, I'd keep up with the project code "syntax". ;)